quadriga/src/main/java/edu/asu/diging/quadriga/core/service/impl/MappedTripleGroupServiceImpl.java

```diff
@@ -144,7 +151,13 @@ public MappedTripleGroup get(String collectionId, MappedTripleType mappedTripleT
         // we create a new MappedTripleGroup entry
         if (mappedTripleGroup == null) {
             mappedTripleGroup = add(collectionId, mappedTripleType);
+            
+            // If mappedTripleGroup add fails
+            if (mappedTripleGroup == null) {
```

Comment:
how can this be null? Wouldn't that only happen if saving the triple group returns null, which I don't think it ever does?